app/routes/collection/add.js

```diff
@@ -2,13 +2,29 @@ import Ember from 'ember';
 
 
 export default Ember.Route.extend({
+
     panelActions: Ember.inject.service('panelActions'),
+    caxe: Ember.inject.service(),
+
     model() {
         const collectionType = this.modelFor('collection').get('collectionType');
-        return Ember.RSVP.hash({
-            workflow: this.store.findRecord('workflow', collectionType),
-            collection: this.modelFor('collection'),
-        });
+        let submissionFormId = {
+            'meeting': 1
+        }[collectionType];
```

Comment:
This is ugly but I was lazy. Need to use queryRecord not findRecord on line 15.

Reply:
I think it might be worth cleaning this up instead of introducing ugly code that we need to fix later then!